
# state of the service
bool working

# failure
bool failure

# goal cancelled
bool goal_cancelled

# the last set target
float32 x
float32 y
float32 th

# if the target has been reached or not
bool success

# time elapsed from the beginning of the mission
float32 time